### PR TITLE
Add regression tests for "helm repo"

### PR DESCRIPTION
**Pull request by marckhouzam**
There were some regressions recently on Helm v3.  They got fixed pretty quickly, but it motivated me to start writing some regression tests.

This PR adds tests to verify that the subcommands of "helm repo" work as expected.

@jdolitsky Of particular interest is the introduction of the Sh class (I am open to other name suggestions).  Being able to test many different helm commands with flag combinations would have required to write **many** python wrappers in the Helm.py class, one for each command.  Instead, I propose the use of the Sh class which allows to run shell commands directly without specific python wrappers.  It also looks pretty cool in the test suite, e.g.,
```
Sh.Run  helm repo list       // to simply run the specified command
Sh.Fail  helm repo add       // to run the command and expect a failure return code
Sh.Pass  kubectl get nodes   // to run the command and expect a successful return code
```

What do you think?

### CODE REVIEW

**Comment by jdolitsky** [comment]
@marckhouzam I like this alot (the Sh class). Not sure how I feel about the name of the .robot file ..  maybe just repos.robot?

File: no_cluster.robot [hunk 1]
```diff
@@ -0,0 +1,79 @@
+*** Settings ***
```
Comment (jdolitsky) [inline no_cluster.robot]: Can we call this file just repos.robot? There might be more repo stuff to test, and also alot more non-cluster stuff that can pushed into other files

Comment (jdolitsky) [inline no_cluster.robot]: Can we call this file just repos.robot? There might be more repo stuff to test, and also alot more non-cluster stuff that can pushed into other files

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: no_cluster.robot (hunk 1 above)
Comment (marckhouzam) [reply no_cluster.robot]: As you deduced, I used that name in preparation to add tests for other non-cluster commands. But I like splitting those into separate files instead. I will rename.

Thanks for all the reviews!

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**